src/telemetry/Opaques.cc

```diff
@@ -0,0 +1,24 @@
+#include "Opaques.h"
+
+using namespace zeek;
+
+TelemetryVal::TelemetryVal(std::shared_ptr<telemetry::Counter>) : OpaqueVal(int_counter_metric_type) {}
```

Comment:
Are these `int_...` types intended Tim, or leftovers from the previous iterations? I did a search over the `src` tree looking for `dbl_` and `int_` and I'm seeing a few more in `zeek-setup.cc` and `Type.h`, such as:
```
Type.h:extern zeek::OpaqueTypePtr dbl_counter_metric_type;
Type.h:extern zeek::OpaqueTypePtr dbl_counter_metric_family_type;
Type.h:extern zeek::OpaqueTypePtr dbl_gauge_metric_type;
Type.h:extern zeek::OpaqueTypePtr dbl_gauge_metric_family_type;
Type.h:extern zeek::OpaqueTypePtr dbl_histogram_metric_type;
Type.h:extern zeek::OpaqueTypePtr dbl_histogram_metric_family_type;
zeek-setup.cc:zeek::OpaqueTypePtr dbl_counter_metric_type;
zeek-setup.cc:zeek::OpaqueTypePtr dbl_counter_metric_family_type;
zeek-setup.cc:zeek::OpaqueTypePtr dbl_gauge_metric_type;
zeek-setup.cc:zeek::OpaqueTypePtr dbl_gauge_metric_family_type;
zeek-setup.cc:zeek::OpaqueTypePtr dbl_histogram_metric_type;
zeek-setup.cc:zeek::OpaqueTypePtr dbl_histogram_metric_family_type;
zeek-setup.cc:    dbl_counter_metric_type = make_intrusive<OpaqueType>("dbl_counter_metric");
zeek-setup.cc:    dbl_counter_metric_family_type = make_intrusive<OpaqueType>("dbl_counter_metric_family");
zeek-setup.cc:    dbl_gauge_metric_type = make_intrusive<OpaqueType>("dbl_gauge_metric");
zeek-setup.cc:    dbl_gauge_metric_family_type = make_intrusive<OpaqueType>("dbl_gauge_metric_family");
zeek-setup.cc:    dbl_histogram_metric_type = make_intrusive<OpaqueType>("dbl_histogram_metric");
zeek-setup.cc:    dbl_histogram_metric_family_type = make_intrusive<OpaqueType>("dbl_histogram_metric_family");
```
(and in `script_opt`, though that'll need a follow-up as usual anyway?)

Reply:
I made the changes in `script_opt`. It probably broke something, but we really need to add some more automated testing for that stuff.